Mathlib/Topology/Algebra/Category/ProfiniteGrp.lean

```diff
@@ -0,0 +1,160 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Nailin Guan, Yuyang Zhao
+-/
+import Mathlib.Algebra.Category.Grp.FiniteGrp
+import Mathlib.Topology.Algebra.ContinuousMonoidHom
+import Mathlib.Topology.Category.Profinite.Basic
+
+
+/-!
+
+# Category of Profinite Groups
+
+We say `G` is a profinite group if it is a topological group which is compact and totally
+disconnected.
+
+## Main definitions and results
+
+* `ProfiniteGrp` is the category of profinite groups.
+
+* `ProfiniteGrp.pi` : The product of profinite groups is also a profinite group.
+
+-/
+
+suppress_compilation
```

Comment:
Why do we need this line?

Reply:
I don't know clearly, it is Yuyang Zhao who added this when refining my codes to make it compile faster. For here maybe I can delete it.